common/base/Cargo.toml

```diff
@@ -24,8 +24,11 @@ common-tracing = { path = "../tracing" }
 async-trait = "0.1.52"
 ctrlc = { version = "3.2.1", features = ["termination"] }
 futures = "0.3.21"
+poem = { version = "1.3.6", features = ["rustls"] }
 pprof = { version = "0.6.2", features = ["flamegraph", "protobuf"] }
+serde = { version = "1.0.136", features = ["derive"] }
+serde_json = { version = "1.0.79", default-features = false, features = ["raw_value"] }
+serde_yaml = { version = "0.8.23", default-features = false }
 tokio = { version = "1.17.0", features = ["full"] }
+toml = { version = "0.5.8", default-features = false }
```

Comment:
Looks tapol fmt not work, how do you format this toml file?

Reply:
I just use make lint and it gives me this order.